### PR TITLE
Reactivate inactive subscribers [MAILPOET-2003]

### PR DESCRIPTION
**Pull request by costasovo**

### CODE REVIEW

File: tests/acceptance/SettingsInactiveSubscribersChangeCest.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    $I->login();
+    $I->amOnMailPoetPage ('Settings');
+    $I->click('[data-automation-id="settings-advanced-tab"]');
+    $I->waitForElement('[data-automation-id="inactive-subscribers-enabled"]');
+    $I->click('[data-automation-id="inactive-subscribers-option-never"]');
+    $I->click('[data-automation-id="settings-submit-button"]');
+    $I->waitForText('Settings saved');
+    $I->amOnMailPoetPage('Subscribers');
+    $I->waitForListingItemsToLoad();
+    // Subscribers are activated in background so we do a couple of reloads
```
[the remark this answers is not in the document]
Comment (costasovo) [reply tests/acceptance/SettingsInactiveSubscribersChangeCest.php]: Unfortunately it is a background task, but it is scheduled to the past so it should run immediately. Locally it passes after 3-4 cycles and on circle it was 6 cycles in every from all 4 runs.

[the remark this answers is not in the document]
Comment (NeosinneR) [reply tests/acceptance/SettingsInactiveSubscribersChangeCest.php]: Ok, we'll see how it'll run in the future. Otherwise looks good.

**Comment by costasovo** [comment]
@michelleshull This activation/inactivation happens in a background job (for performance reasons). Initially this background job runs once a week. With this change it is scheduled to start right after setting is changed but it still may take a up to a minute (depends on number of subscribers) to complete. What was the final result after some time?

**Comment by michelleshull** [comment]
Retested, and you're correct, it took about a minute, but all the inactives were made subscribed again after I let it sit for a little bit. 
We should note this to support so they can add it to the kb, in case users are as hasty as I was. @bruberries

**Comment by costasovo** [comment]
@michelleshull We don't say anywhere that this is instant, but I agree that it would be better to note that this not instant action but it is done regularly in a background.

**Pull request merged**